tests/integration/update_cluster/existing_iam/in-v1alpha2.yaml

```diff
@@ -64,6 +68,26 @@ spec:
 
 ---
 
+apiVersion: kops/v1alpha2
```

Comment:
This would actually cause the test to fail, Kops requires an odd number of masters. So when testing an HA setup we;d need at least 3 masters defined.